internal/db/migrations/2025-06-10.4-create_indexer_table_state_changes.sql

```diff
@@ -9,8 +9,8 @@ CREATE TABLE state_changes (
     ledger_created_at TIMESTAMPTZ NOT NULL,
     ledger_number INTEGER NOT NULL,
     account_id TEXT NOT NULL REFERENCES accounts(stellar_address),
-    operation_id BIGINT NOT NULL REFERENCES operations(id),
-    tx_hash TEXT NOT NULL REFERENCES transactions(hash),
```

Comment:
Shouldn't we keep referencing the tx_hash here?

Reply:
@marcelosalloum So we are running into an issue here where state changes for contract invocations are processed but we did not insert those transactions because we dont identify the contract participants correctly. So this is leading to an error when we insert the state change but the txn hash has not been inserted.

And for the operation ID, fee events dont have an operation associated with them so we get an error when inserting those fee state changes